src/Api/AbstractApi.php

```diff
@@ -36,7 +36,7 @@ abstract class AbstractApi
      *
      * @var string
      */
-    private const URI_PREFIX = '/api/v4/';
+    protected const URI_PREFIX = '/api/v4/';
```

Comment:
Maybe we should split version out of `URI_PREFIX`, into a protected class? Then we override the version in GraphQL as GitLab GraphQL is versionless